src/games/strategy/triplea/attachments/TriggerAttachment.java

```diff
@@ -2603,9 +2632,11 @@ public static void triggerVictory(final Set<TriggerAttachment> satisfiedTriggers
   }
 
   // All triggers can be activated in only 1 of 2 places: default or when
-  // default = t.getWhen.isEmpty() (this means when was not set, and so the trigger should activate in its default place, like before
+  // default = t.getWhen.isEmpty() (this means when was not set, and so the trigger should activate in its default
```

Comment:
Inappropriate location for the comment, and a bit nonsensical - what does "t.getWhen().isEmpty()" mean?? Does that still compile even? These comments do not seem to add value, look like they shoudl be axed or cleaned up.